sw/device/silicon_creator/manuf/lib/individualize.c

```diff
@@ -23,7 +23,7 @@
 
 #include "otp_ctrl_regs.h"
 
-typedef struct hw_cfg0_settings {
+typedef struct hw_cfg1_settings {
```

Comment:
@timothytrippel can you have a look at these individualize / personalize test changes and see whether they make sense?